### PR TITLE
Devsite v7 22

### PR DESCRIPTION
**Pull request by udaymattam**
### Summary

Changes in terra-dev-site were introduced starting in v7, and persisted in v8, that caused clipping issues for some dev-site testing pages.
### Deployment Link

https://terra-applic-devsite-v7-cirynp.herokuapp.com/tests/terra-application/application-navigation/private/application-navigation/application-navigation
### Additional Details

Thank you for contributing to Terra.
@cerner/terra

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -83,7 +83,7 @@
     "wdio-default": "terra wdio",
     "wdio-lowlight": "terra wdio --themes clinical-lowlight-theme",
     "wdio-fusion": "terra wdio --themes orion-fusion-theme",
-    "wdio": "terra wdio --themes terra-default-theme clinical-lowlight-theme orion-fusion-theme"
+    "wdio": "terra wdio --themes terra-default-theme"
```
Comment (supreethmr) [inline package.json]: need to revert this changes !!

**Review by supreethmr — APPROVED** [review]

**Review by neilpfeiffer — APPROVED** [review]
Tested: PR #310
Resolves: UXPLATFORM-6906

**Reviewed:**
- Correctly resolves problems with dev-site page clipping, ports fixes backs to dev-site v7.
- Can see similar information on v8 approval: https://github.com/cerner/terra-application/pull/308#pullrequestreview-1119140078

 **UX Reviewed** |
-----------|

**Review by saket2403 — APPROVED** [review]
